docs/internals/deprecation.txt

```diff
@@ -30,6 +30,10 @@ details on these changes.
 * The ``extra_tests`` argument for ``DiscoverRunner.build_suite()`` and
   ``DiscoverRunner.run_tests()`` will be removed.
 
+* The ``django.contrib.postgres.aggregates.ArrayAgg``, ``JSONBAgg``, and
+  ``StringAgg`` aggregates will return ``None`` when there are no rows instead
+  of ``[]``, ``[]``, and ``''`` respectively.
```

Comment:
I changed `'[]'` to `[]` because `JSONBAgg` doesn't return strings.

Reply:
Thanks. I mixed this up with the value returned in `convert_value()` that we continue to use for `deprecation_value` as that happens before the JSON is deserialised.